packages/teleterm/src/mainProcess/mainProcessClient.ts

```diff
@@ -32,10 +32,10 @@ export default function createMainProcessClient(): MainProcessClient {
       return ipcRenderer.invoke('main-process-force-focus-window');
     },
     symlinkTsh() {
```

Comment:
I somehow forgot to send a comment earlier, but I'd also add `macOs` part to `symlinkTsh`/`removeTshSymlink`.